app/ui/src/test/java/com/fsck/k9/activity/setup/AccountDiscoveryTest.kt

```diff
@@ -0,0 +1,111 @@
+package com.fsck.k9.activity.setup
+
+import com.fsck.k9.RobolectricTest
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertNotNull
+import org.junit.Test
+
+class AccountDiscoveryTest : RobolectricTest() {
```

Comment:
Some tests with malformed input would be nice.

Reply:
Added two tests with broken XML or missing required settings.